src/main/scala/mesosphere/marathon/api/v2/json/V2AppDefinition.scala

```diff
@@ -2,17 +2,23 @@ package mesosphere.marathon.api.v2.json
 
 import java.lang.{ Double => JDouble, Integer => JInt }
 
+import com.wix.accord._
 import mesosphere.marathon.Protos.Constraint
+import mesosphere.marathon.Protos.HealthCheckDefinition.Protocol
+import mesosphere.marathon.api.v2.Validation._
 import mesosphere.marathon.api.validation.FieldConstraints._
 import mesosphere.marathon.api.validation.{ PortIndices, ValidV2AppDefinition }
 import mesosphere.marathon.health.HealthCheck
 import mesosphere.marathon.state.AppDefinition.VersionInfo.FullVersionInfo
 import mesosphere.marathon.state._
+import mesosphere.marathon.state.PathId._
 import org.apache.mesos.{ Protos => mesos }
 
 import scala.collection.immutable.Seq
 import scala.concurrent.duration._
 
+import com.wix.accord.dsl._
+
 @PortIndices
```

Comment:
From my point of view we do. Let me give an example in the mentioned class:
```scala
  implicit val appDefinitionValidator = validator[V2AppDefinition] { appDef =>
    appDef.id is valid
    appDef.dependencies is valid
    appDef.upgradeStrategy is valid
    appDef.storeUrls is every(urlsCanBeResolvedValidator)
    appDef.ports.filterNot { _ == AppDefinition.RandomPortValue } as "ports" is elementsAreUnique
    appDef.executor should matchRegexFully("^(//cmd)|(/?[^/]+(/[^/]+)*)|$")
    appDef is containsCmdArgsContainerValidator
    appDef is portIndicesAreValid
    appDef.instances.intValue should be >= 0
  }
```
Here we have a predefined validator `matchRegexFully`. It could of course be written by us as well, but I found it convenient to use the provided one.
Functions like `is` or `should` are also part of the package. In order to define our class validators, we need to import that package. An alternative would be defining that class validators outside of the class file. We decided to do it this way back then, but I don't mind thinking about alternatives again.